Apps/RaptureIntTests/src/test/resources/local_testng.xml

```diff
@@ -13,6 +13,9 @@
 			<package name="rapture.httpapi.series" />
 			<package name="rapture.httpapi.document" />
 			<package name="rapture.httpapi.plugin" />
+			<package name="rapture.nightly.java" />
```

Comment:
a package name - we haven't formalized naming yet.
There's also rapture.nightly.java.search and rapture.nightly.reflex (stuff needed for Reflex testing)

Jeremy has been using rapture.httpapi.{apiname}